sdk/core/core/src/az_http_policy_logging.c

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "az_http_policy_logging_private.h"
+
+#include "az_log_private.h"
+#include <az_clock_internal.h>
+#include <az_http_response_parser.h>
+#include <az_log.h>
+#include <az_log_internal.h>
+#include <az_span_builder.h>
+#include <az_str.h>
+
+#include <stdbool.h>
+#include <stddef.h>
+
+#include <_az_cfg.h>
+
+enum {
+  _az_LOG_VALUE_MAX_LENGTH
+  = 50, // When we print values, such as header vaules, if they are longer than
```

Comment:
Placing the comment above may look better and render much better search results in most IDEs which display a single line (no context) when displaying search results.